tests/manage/test_create_storageclass_with_same_name.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import pytest
+
+from ocs import defaults
+from ocsci.testlib import tier1, ManageTest
+from resources.ocs import OCS
+from ocs.exceptions import CommandFailed
+from ocsci.config import ENV_DATA
+
+log = logging.getLogger(__name__)
+
+SC_OBJ = None
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    This fixture defines the teardown function.
+    """
+
+    request.addfinalizer(teardown)
+
+
+def teardown():
+    """
+    Tearing down the environment
+
+    """
+    log.info(f"Deleting created storage class: {SC_OBJ.name}")
+    SC_OBJ.delete()
+    log.info(f"Storage class: {SC_OBJ.name} deleted successfully")
+
+
+def create_storageclass(sc_name, expect_fail=False):
+    """
+    Function to create a storage class and check for
+    duplicate storage class name
+
+    Args:
+        sc_name (str): name of the storageclass to be created
+        expect_fail (bool): To catch the incorrect scenario if
+            two SCs are indeed created with same name
+
+    Returns:
+        None
+
+    """
+
+    # Create a storage class
+    mons = (
+        f'rook-ceph-mon-a.{ENV_DATA["cluster_namespace"]}'
+        f'.svc.cluster.local:6789,'
+        f'rook-ceph-mon-b.{ENV_DATA["cluster_namespace"]}.'
+        f'svc.cluster.local:6789,'
+        f'rook-ceph-mon-c.{ENV_DATA["cluster_namespace"]}'
+        f'.svc.cluster.local:6789'
+    )
+    log.info("Creating a Storage Class")
+    sc_data = defaults.CSI_RBD_STORAGECLASS_DICT.copy()
+    sc_data['metadata']['name'] = sc_name
+    sc_data['parameters']['monitors'] = mons
+
+    global SC_OBJ
+    SC_OBJ = OCS(**sc_data)
+
+    # Check for expected failure with duplicate SC name
+    try:
+        SC_OBJ.create()
+        assert not expect_fail, (
+            "SC creation with same name passed. Expected to fail !!"
+        )
+        log.info(f"Storage class: {SC_OBJ.name} created successfully")
+        log.debug(sc_data)
+
+    except CommandFailed as ecf:
+        assert "AlreadyExists" in str(ecf)
+        log.error(f"Cannot create two StorageClasses with same name !! \n"
```

Comment:
fix parentheses:

```python 
log.error(
    f"Cannot create two StorageClasses with same name !! {ecf}"
)
```

Please apply to all relevant places

Reply:
@ebenahar ok. Working on it

Reply:
@ebenahar Do I need to do this indentation even if the logged info is short ?

Reply:
@ebenahar made the changes... To the best of my understanding :) 

Please review

Reply:
@nehaberry you should split to a new line in case it exceeds 80 chars. Now it's looking good, no need for further change